app/forms/idv/phone_form.rb

```diff
@@ -79,6 +79,7 @@ def validate_phone_delivery_methods
             "two_factor_authentication.otp_delivery_preference.#{delivery_method}_unsupported",
             location: capabilities.unsupported_location,
           ),
+          type: :phone,
```

Comment:
```suggestion
          type: :unsupported_location,
```
might want to include delivery method too in there:
```suggestion
          type: :"#{delivery_method}_unsupported",
```